resources_pages/install_ds_stack_windows.md

```diff
@@ -3,290 +3,543 @@ layout: page
+
+![](/resources_pages/imgs/vs_code.png)
+
 ## GitHub
 
-In MDS we will use [GitHub.com](https://github.com/) as well as an Enterprise version of GitHub hosted here at UBC, [GitHub.ubc.ca](https://github.ubc.ca). Please follow the set-up instructions for both below.
+In MDS we will use the publicly available [GitHub.com](https://github.com/) as well as an Enterprise version of GitHub hosted here at UBC, [GitHub.ubc.ca](https://github.ubc.ca). Please follow the set-up instructions for both below.
+
+### GitHub.com
 
-#### GitHub.com
-If you do not yet have one, sign up for a free account at https://github.com/ .
+Sign up for a free account at [GitHub.com](https://github.com/) if you don't have one already.
 
-#### GitHub.ubc.ca
-For us to add you to the MDS organization on [Github.ubc.ca](https://github.ubc.ca) we need you to login using your CWL:
+### GitHub.ubc.ca
 
-visit [Github.ubc.ca](https://github.ubc.ca) to do this.
+To add you to the MDS organization on [Github.ubc.ca](https://github.ubc.ca) we need you to login to [Github.ubc.ca](https://github.ubc.ca) using your CWL credentials.
 
 This step is required for
 - being able to store your work
 - all homework submission and grading
 - working collaboratively
 
-## Git
+## Git and Bash
+
+These two programs are in the same section since they are packaged together in the same installer.
+Briefly, we will be using the Bash shell to interact with our computers via a command line interface.
+We will be using the command line version of Git as well as Git through RStudio and Jupyter lab. Some of the Git commands we will use are only available since Git 2.23, so if you're Git is older than this version, we ask you to update it.
 
-We will be using the command line version of Git as well as Git through RStudio. 
+Go to <https://git-scm.com/download/win> and download the windows version of git. After the download has finished, run the installer and accept the default configuration for all pages except for the following:
+    - *Optional* On the **Select Components** page, check "On the Desktop" under "Additional icons".
+    - On the **Choosing the default editor used by Git** page, select "Use Visual Studio Code as Git's default editor" from the drop-down menu'
 
-Go to <https://git-scm.com/download/win> and a download should automatically start (if not, follow the instructions on that page). After the download has finished, run the installer selecting the following options:
-  - On the **Select Components** page, check "On the Desktop" under "Additional icons"
-  - On the **Choosing the default editor used by Git** page, select "Use the Nano editor by default" from the drop-down menu
-  - For all other pages, use the default options
+> Note that we don't recommend pinning Git Bash to the taskbar since launches Git Bash from here will set a different home directory than if Git Bash is launched from the desktop or the start menu (`/` instead of `/c/users/$USERNAME`.
```

Comment:
Things like this makes me want to force everyone to dual boot Ubuntu...

Reply:
@ttimbers I found the solution for this here https://github.com/git-for-windows/git/issues/1790#issuecomment-412493650

> That happens because Git GUI is a script and when you try to pin a running script to the taskbar, windows pins the executable that runs the script instead. Navigate to Git GUI in your start menu, right click the entry and select "pin to taskbar" to actually pin Git GUI (well, the GfW wrapper that launches Git GUI) to your taskbar.

Will update this note.